HoloPipeLines-Models/dense_vnet_abdominal_ct/README.md

```diff
@@ -0,0 +1,73 @@
+# Niftynet abdominal neural network container
+This page describes how to create a docker image that is with the endpoint for the network described in
+
+Eli Gibson, Francesco Giganti, Yipeng Hu, Ester Bonmati, Steve Bandula, Kurinchi Gurusamy, Brian Davidson, Stephen P. Pereira, Matthew J. Clarkson and Dean C. Barratt (2017), Automatic multi-organ segmentation on abdominal CT with dense v-networks https://doi.org/10.1109/TMI.2018.2806309
+
+This network model container segments eight organs on abdominal CT, comprising the gastointestinal tract (esophagus, stomach, duodenum), the pancreas, and nearby organs (liver, gallbladder, spleen, left kidney).
+
+Here is the repo for the network model
+https://github.com/NifTK/NiftyNetModelZoo/tree/5-reorganising-with-lfs/dense_vnet_abdominal_ct
+the repo only runs in the niftynet framework so to run the network model locally you need to follow the isntruction from the niftynet and download the framework first
+https://niftynet.readthedocs.io/en/dev/installation.html
+
+
+## Build Docker image from the Docker file
+
+```
+docker build -t [docker name:tag] .
+# example
+docker build -t niftynet:latest .
+```
+
+This instruction will create the niftynet docker image with flask server end point. to run the image you create you need to run the other command
+
+
+## run docker and connect docker to the port 5000 
+```
+docker run -d -p 5000:5000 [docker name]
```

Comment:
Specify 2 additional flags `--name="niftynet"` and `--rm`. 

--name because we can use it later for testing.

--rm because the container will be removed after the image is closed without leaving it dangling.